src/main/java/com/vegaone/venus/domain/CompanyEntity.java

```diff
@@ -25,4 +25,6 @@ public class CompanyEntity {
             orphanRemoval = true,
             fetch = FetchType.EAGER)
     private List<ProjectEntity> projects;
+    @OneToOne(mappedBy = "company")
```

Comment:
We need a `@OneToMany` instead of `@OneToOne`, since one company can have many users. It's strange that the `shouldHaveTwoUsersAssignedToCompany` test passes.